README.md

```diff
@@ -3,13 +3,17 @@
 [![Build Status](https://travis-ci.com/laminas/laminas-skeleton-installer.svg?branch=master)](https://travis-ci.com/laminas/laminas-skeleton-installer)
 [![Coverage Status](https://coveralls.io/repos/github/laminas/laminas-skeleton-installer/badge.svg?branch=master)](https://coveralls.io/github/laminas/laminas-skeleton-installer?branch=master)
 
-laminas-skeleton-installer is a composer plugin for use in the initial install of
-the [laminas-mvc-skeleton](https://github.com/laminas/laminas-mvc-skeleton).
-It prompts for common requirements, adding packages to the composer requirements
-for each selection, and then uninstalls itself on completion.
+laminas-skeleton-installer is a composer plugin for use in the initial install 
+of the [laminas-mvc-skeleton](https://github.com/laminas/laminas-mvc-skeleton).
+It prompts for common requirements, adding packages to the composer 
+requirements for each selection, and then uninstalls itself on completion.
 
 The installer requires [laminas-component-installer](https://docs.laminas.dev/laminas-component-installer/),
 and we recommend requiring that component in your project skeleton as well.
 
+**Please note:** this repository is a helper to the 
+[laminas-mvc-skeleton](https://github.com/laminas/laminas-mvc-skeleton) 
+repository and is not intended to be used on its own.
+
```

Comment:
```suggestion
**Please note:** this repository is a **composer plugin** for use in skeleton
applications (such as [laminas-mvc-skeleton](https://github.com/laminas/laminas-mvc-skeleton)), 
and is not intended to be installed as a general-purpose package.

```

Its use is not limited to laminas-mvc-skeleton. We actually briefly used it in the Expressive and Apigility skeletons (though the needs of the former required a different approach eventually, and the latter, being highly specialized, has already answered the various "questions" the MVC skeleton asks). As such, I'm making the above suggestion to make that more clear, as well as highlighting that this is a composer plugin, and not a general-purpose package.